pages/item/_id.vue

```diff
@@ -26,17 +26,18 @@
         </div>
 
         <div v-if="isLocal" class="flex mt-4 -mr-2">
-          <ui-btn color="success" :disabled="isPlaying" class="flex items-center justify-center flex-grow mr-2" :padding-x="4" @click="playClick">
+          <ui-btn color="success" :disabled="isPlaying" small class="flex flex-grow justify-center items-center h-9 mr-2" :padding-x="4" @click="playClick">
             <span v-show="!isPlaying" class="material-icons">play_arrow</span>
             <span class="px-1 text-sm">{{ isPlaying ? 'Playing' : 'Play Local' }}</span>
           </ui-btn>
           <ui-btn v-if="showRead && isConnected" color="info" class="flex items-center justify-center mr-2" :class="showPlay ? '' : 'flex-grow'" :padding-x="2" @click="readBook">
             <span class="material-icons">auto_stories</span>
             <span v-if="!showPlay" class="px-2 text-base">Read {{ ebookFormat }}</span>
           </ui-btn>
+          <ui-read-icon-btn :disabled="isProcessingReadUpdate" :is-read="userIsFinished" class="mx-0.5 flex items-center justify-center" @click="toggleFinished" />
```

Comment:
We can't show this for podcasts

Reply:
I didn't think this page was accessible for a podcast item, they have their own page with independent buttons to mark as read correct? But if this does ever get used for a podcast I added `v-if="!isPodcast"`. Was that what you meant?